packages/eslint-plugin/src/rules/explicit-function-return-type.ts

```diff
@@ -139,6 +146,16 @@ export default util.createRule<Options, MessageIds>({
       }
       return false;
     }
+
+    function isIIFE(
+      node: TSESTree.ArrowFunctionExpression | TSESTree.FunctionExpression,
+    ): boolean {
+      return (
+        node.parent?.type === AST_NODE_TYPES.CallExpression &&
+        node.parent.callee === node
```

Comment:
Can I add coverage ignoring comment here? (`istanbul ignore next`) or Maybe this line is useless?

Reply:
If code coverage is lacking on a line, one of the two following things is true:

* 🍎 The line is possible to reach and so should be unit tested
* 🍌 The line is impossible to reach and so should ideally be removed

If you can't unit test it (🍎), then it should be removed (🍌).